docs/overview/configuration.md

```diff
@@ -70,6 +70,44 @@ example.com
 *example.com
 ```
 
+### Intercepted Resources
+
+This allows you to intercept resources and modify them before they are loaded. This is useful for modifying the response of a request.
+
+```js
+const hero = new Hero({
+  interceptedResources: [
+    {
+      url: 'https://example.com',
+      body: '<div>Hello World!</div>',
+    },
+  ],
+});
+
+const heroStatus = new Hero({
+  interceptedResources: [
+    {
+      url: 'https://example.com',
+      statusCode: 404,
+    },
+  ],
+});
+
+const heroHeaders = new Hero({
+  interceptedResources: [
+    {
+      url: 'https://example.com',
+      headers: [
```

Comment:
Will this add the header but still reach out to the upstream server?

Reply:
I was wondering if you were expecting this to simply "add a header" but still fulfill the request to 'http://example.com'

Reply:
This code is supposed to change the headers of the response, not the request.

Reply:
😵‍💫 Oh man. Yes, you are correct. Brain fart.